Mathlib/Tactic/Linter/FlexibleLinter.lean

```diff
@@ -291,13 +299,29 @@ def flexible : Std.HashSet Name :=
     ``Lean.Parser.Tactic.tacticRfl,
     ``Lean.Parser.Tactic.omega,
     `Mathlib.Tactic.Abel.abel,
+    `Mathlib.Tactic.Abel.tacticAbel!,
+    `Mathlib.Tactic.Group.group,
     `Mathlib.Tactic.RingNF.ring,
+    `Mathlib.Tactic.RingNF.tacticRing!,
+    `Mathlib.Tactic.Module.tacticModule,
+    `Mathlib.Tactic.Positivity.positivity,
+    ``Lean.Parser.Tactic.grind,
     `Mathlib.Tactic.normNum,
-    `linarith,
-    `nlinarith,
+    `Mathlib.Tactic.linarith,
+    `Mathlib.Tactic.nlinarith,
+    `Mathlib.Tactic.tacticNlinarith!_,
     `Mathlib.Tactic.LinearCombination.linearCombination,
     ``Lean.Parser.Tactic.tacticNorm_cast__,
     `Aesop.Frontend.Parser.aesopTactic,
+    -- `cfc_tac` and `cfc_zero_tac` use `aesop` under the hood, `cfc_cont_tactic` also uses `fun_prop`.
+    `cfcTac,
+    `cfcZeroTac,
+    `cfcContTac,
+    -- `continuity` and `measurability` also use `aesop` under the hood.
+    `tacticContinuity,
+    `tacticMeasurability,
```

Comment:
Are `continuity` and `measurability` tested below?  I wonder mostly because these are the ones that stand out as not having their name checked here, and so could become silently unchecked.